### PR TITLE
Mdx2 upgrade v3

### PR DESCRIPTION
**Pull request by Aviral0702**
**Description**

This PR fixes #
Aims to migrate to MDX V2 and gatsby-plugin-mdx v4

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
where are the other build checks here?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
why there is changes in md and yml files , i don't understand?

**Comment by sudhanshutech** [comment]
@Aviral0702 will you explain the breaking changes here while migarte.

**Comment by sudhanshutech** [comment]
please reply on all of the questions asked above .

**Comment by sudhanshutech** [comment]
> > why there is changes in md and yml files , i don't understand?
> 
> The older configurations of mdx files are not compatible with new rules of parsing mdx files in MDX V2. That is why we need to change the configuration here and if it works we will permanently change the configuration of these files.

is it? i still don't understand why yml files are been changed here

**Comment by Aviral0702** [comment]
> @Aviral0702 merge conflicts

resolved.

**Comment by vishalvivekm** [comment]
@Aviral0702 @sudhanshutech 
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by vishalvivekm** [comment]
I've added this to the agenda items @sudhanshutech @Aviral0702

File: .github/workflows/lighthouseci.yml
Comment (leecalcote) [inline .github/workflows/lighthouseci.yml:1]: I'm missing it. What changed in these files?

**Comment by vishalvivekm** [comment]
@Aviral0702 
Are we discussing updates on this PR today on websites' call ?

**Pull request closed without merging**